extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -79,12 +84,18 @@ export function RemoteQueries(): JSX.Element {
       </div>
 
       <ul className="vscode-codeql__results-list">
-        {queryResult.results.map(result =>
+        {queryResult.results.slice(0, numOfReposToShow).map((result, i) =>
```

Comment:
Can we ever get in a situation with 0 results? If so, maybe we want to show a special message here.

Not something that is critical right away, but something to think about.

Reply:
That is a great point - I'll raise a UX question!